static/js/modules/helpers.js

```diff
@@ -216,6 +215,41 @@ function zeroPad(container, item, appendee) {
   });
 }
 
+function amendmentVersion(data) {
```

Comment:
I changed the variable to the more descriptive (and what the parameter is actually returned from API) `most_recent`.